TCMP/BasicNFC-CommandFamily/BasicNFCCommandResolver.swift

```diff
@@ -77,4 +77,13 @@ import Foundation
         try parsedMessage.parsePayload(payload: message.payload)
         return parsedMessage
     }
+    
```

Comment:
I will add a proper NDEF library to this SDK in the future, but this cannot be added here.  I know I did something like that for the illustrative app I sent you privately, but there are a lot of reasons why this approach is not robust enough for the core SDK.

Reply:
Yea I just copied that method from your demo app, didn't even really look into it. I'll update this method to just return the raw NDEF message and I'll do the parsing on the app-side until you have that NDEF library in. I do need a method like this one in order to get the NDEF data (tagCode and ndef text) to be in the core SDK as I cannot directly access properties of the NDEF object on the NativeScript side. I'm gonna modify it so it returns a JSON object with the tag code and the full ndef